csharp/lib/src/connection.rs

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+ */
+use std::ffi::c_char;
+
+#[repr(C)]
+pub struct ConnectionConfig {
```

Comment:
Should we have documentation for the types here?

Reply:
1:1
Will do

Reply:
Done. Do you know how to add a reference to an arbitrary file by relative path ?

Reply:
You mean like this? https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file

Reply:
I can't understand where relative path starts from. I tried to point to `connection_request.proto`, but all experiments with ../../.. failed